charts/stable/channels-dvr/values.yaml

```diff
@@ -0,0 +1,27 @@
+image:
+  repository: tccr.io/truecharts/channels-dvr
+  pullPolicy: Always
+  tag: latest@sha256:5cad00679feb37533f0eccf72b70c3bf7acc4b4c18fcae65e2081e7222da39ce
```

Comment:
use this instead
```yaml
image:
  repository: fancybits/channels-dvr #tccr.io/truecharts/channels-dvr
  pullPolicy: Always
  tag: latest@sha256:5cad00679feb37533f0eccf72b70c3bf7acc4b4c18fcae65e2081e7222da39ce
```

Reply:
@fenichelar Are you really wanna use an app that hasn't released an image for over 2 years? :/

Reply:
@stavros-k It appears that the application updates itself when setup. I am currently using `custom-app` to run the container and the application reports it is on version `2022.11.15.1908`.

Reply:
@Xstar97 Done in https://github.com/truecharts/charts/pull/4527/commits/d16faf0e594a9d1b96079f2f82f547fce86c3aa5

Reply:
@Xstar97 I removed the comment with the tccr.io link. The linter was complaining and I couldn't figure out how it wanted the comment to be formatted.